spec/support/factory_girl.rb

```diff
@@ -0,0 +1,3 @@
+RSpec.configure do |config|
+  config.include FactoryGirl::Syntax::Methods
```

Comment:
This is provided by FactoryGirl, it gives us a nice sugar to use in our test.

So instead of the tedious `FactoryGirl.create(....)`

we get to use `create(....)`.